content/6_Advanced/BCC_2CC.mdx

```diff
@@ -178,14 +178,165 @@ struct TwoEdgeCC {
 
 <Problems problems="bccSam" />
 
-note that BCCs contain EDGES not VERTICES
+An important observation is that if you can't go from node `a` to node `b` without passing through node `c`, it means that node `c` is a critical node (articulation point).
+Node `c` can split `a` and `b` into 2 different components if removed. This makes us think about biconnected components.
```

Comment:
use latex

Reply:
What do you mean ?

Reply:
dollar signs instead of the inline code thing

Reply:
Alright done